scss/grid/_gutter.scss

```diff
@@ -90,3 +90,37 @@
   @warn 'This mixin is being replaced by grid-col-gutter(). grid-col-uncollapse() will be removed in Foundation 6.4.';
   @include grid-column-uncollapse($gutter);
 }
+
+/// Sets bottom marin on block grid columns to match gutters
+/// @param {Number|Keyword} $margin [auto]
+///   The bottom margin on block grid columns, accepts multiple values:
+///   - A single value will make the margin that exact size.
+///   - A breakpoint name will make the margin the corresponding size in the $gutters map.
+///   - "auto" will make the margin responsive, using the $gutters map values.
```

Comment:
I see you use "single value", "breakpoint name" and "auto" like `grid-column-gutter`. I searched a way to factorize this into one mixin used by `grid-column-gutter` and `block-grid-column`, but I didn't find it. :/

Also, it's `$margins` and not `$gutters` in `in the $gutters map` and `using the $gutters map values`. Or rename the variable to `$gutters`. After all, it's vertical gutters.